schemas/stsci.edu/asdf/unit/quantity-1.1.0.yaml

```diff
@@ -0,0 +1,27 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/unit/quantity-1.1.0"
+
+title: >
+  Represents a Quantity object from astropy
+
+anyOf:
+  - tag: "tag:stsci.edu:asdf/unit/quantity-1.1.0"
+  - {}
+
+type: object
+properties:
+  value:
```

Comment:
Yes, that makes sense. I'm not sure whether `ndarray` is automatically handled or not but I'll investigate.